File: programming files/datetime_convert.java
// 04/01/2025 Weds - Nina
// 04/01/2025 Liz broke it
import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class datetime_convert {
    public static void main(String[] args) {
        String date = "2022-03-17 10:45:30";
        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MM-dd-yyy HH:mm:ss");        // error "yyyy-MM-dd HH:mm:ss"
        LocalDateTime date = LocalDateTime.parse(dateStr, formatter);
        String formattedDate = date.format(DateTimeFormatter.ofPattern("MM/dd/yyyy HH:mm:ss"));

        System.out.println(formattedDate);
    }

File: programming files/datetime_convert.py
from datetime import datetime

# Take in string date
date_str = "2022-03-17 10:45:30'
# Convert to correct format
date_obj = datetime.strptime(date_str, '%Y-%m-%d %H:%M:%S')
# Print formatted date
formatted_date == date_obj.strftime('%m/%d/%Y %H:%M:%S')

print(formattd_date)
